docs-mslearn/toolkit/help/data-dictionary.md

```diff
@@ -39,169 +39,181 @@ Quick links:
-| ToolkitTool                              | Derived. Name of the tool in the FinOps toolkit the resource supports.                                                                                                                                                                                                                                                  |
-| ToolkitVersion                           | Derived. Version of the tool in the FinOps toolkit the resource supports.                                                                                                                                                                                                                                               |
-| TotalSavings                             | Derived. Used in Power BI to sum the total amount saved after negotiated and commitment discounts are applied. Will be negative for unused commitments. Formula: `ListCost - EffectiveCost`.                                                                                                                            |
-| TotalSavingsRunningTotal                 | Derived. Calculates the accumulated or running total of DiscountCostSavings for the day, including all previous day's values.                                                                                                                                                                                           |
-| <a name="u"></a>UsageCPUHours            | See [`ConsumedCoreHours`](#c).                                                                                                                                                                                                                                                                                          |
-| UsageQuantity                            | Number of units of a resource or service that got used or purchased based on the `UsageUnit`. Replaced by `ConsumedQuantity` in v0.4 to align with FOCUS 1.0.                                                                                                                                                           |
-| UsageUnit                                | Indicates what measurement type is used by the `UsageQuantity`. Replaced by `ConsumedUnit` in v0.4 to align with FOCUS 1.0.                                                                                                                                                                                             |
+| Name                                             | Description                                                                                                                                                                                                                                                                                                             |
```

Comment:
Automatic formatting was applied that changed whitespace. Use the gear icon to hide whitespace changes so it's easier to see what actually changed in this table.